### PR TITLE
Resolve ansible-lint failures

### PR DESCRIPTION
**Pull request by crosscodr**
Main part is to move "set bbb hostname" to handlers.
I tested changing the hostname and also the "restart kurento" handler without the "ignore_errors: true"

### CODE REVIEW

**Comment by BrutalBirdie** [comment]
Hey @crosscodr can you update this PR?
I also just added the `.ansible-lint` so there is now a conflict.
Thanks ahead!

File: .ansible-lint [hunk 1]
```diff
@@ -0,0 +1,2 @@
+warn_list:
+  - package-latest
```
Comment (BrutalBirdie) [inline .ansible-lint:2]:
```suggestion
# .ansible-lint
exclude_paths:
  - .cache/  # implicit unless exclude_paths is defined in config
  - .github/
warn_list:
  - package-latest
```

**Review by BrutalBirdie — CHANGES_REQUESTED** [review]

**Comment by crosscodr** [comment]
Done :)
Will you take my version?

**Review by BrutalBirdie — APPROVED** [review]

**Comment by BrutalBirdie** [comment]
Why does github still nag about:
```
Conflicting files
.ansible-lint
```

**Comment by crosscodr** [comment]
> Why does github still nag about:
> 
> ```
> Conflicting files
> .ansible-lint
> ```

Because I didn't merge it optimal :)

**Comment by BrutalBirdie** [comment]
Yes true that.
I am testing this PR and deploying against the public testing server.

https://short.cloudron.dev/ebbba-org_ansible-role-bigbluebutton_public-testing

**Review by Davka — APPROVED** [review]

**Pull request merged**